packages/tenant-process/test/revokeVerifiedAttribute.test.ts

```diff
@@ -0,0 +1,327 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable @typescript-eslint/no-floating-promises */
+import {
+  generateId,
+  Tenant,
+  unsafeBrandId,
+  protobufDecoder,
+  toTenantV2,
+  Descriptor,
+  EService,
+  descriptorState,
+  tenantAttributeType,
+  AttributeId,
+  TenantVerifiedAttributeRevokedV2,
+  Agreement,
+  toReadModelEService,
+  toReadModelAgreement,
+} from "pagopa-interop-models";
+import { describe, it, expect, vi, afterAll, beforeAll } from "vitest";
+import { genericLogger } from "pagopa-interop-commons";
+import {
+  writeInReadmodel,
+  readLastEventByStreamId,
+} from "pagopa-interop-commons-test";
+import {
+  tenantNotFound,
+  attributeAlreadyRevoked,
+  attributeNotFound,
+  attributeRevocationNotAllowed,
+  verifiedAttributeSelfRevocation,
+} from "../src/model/domain/errors.js";
+import {
+  addOneTenant,
+  getMockAgreement,
+  getMockTenant,
+  getMockEService,
+  getMockDescriptor,
+  getMockVerifiedTenantAttribute,
+  getMockVerifiedBy,
+  getMockRevokedBy,
+  eservices,
+  agreements,
+  tenantService,
+  postgresDB,
+} from "./utils.js";
+
+describe("revokeVerifiedAttribute", async () => {
+  const targetTenant: Tenant = getMockTenant();
+  const requesterTenant: Tenant = getMockTenant();
+  const attributeId: AttributeId = generateId();
```

Comment:
```suggestion
  const attribute = getMockVerifiedTenantAttribute();
```
Since you will always need it maybe it's better to define the attribute here and then you can share its attributeId where needed